integration-tests/src/test/java/oracle/kubernetes/operator/ITOperatorLifecycle.java

```diff
@@ -0,0 +1,196 @@
+  // set INGRESSPERDOMAIN to false to create LB's ingress by kubectl yaml file
+  static {
+    QUICKTEST =
+        System.getenv("QUICKTEST") != null && System.getenv("QUICKTEST").equalsIgnoreCase("true");
+    SMOKETEST =
+        System.getenv("SMOKETEST") != null && System.getenv("SMOKETEST").equalsIgnoreCase("true");
+    if (SMOKETEST) QUICKTEST = true;
+    if (System.getenv("JENKINS") != null) {
+      JENKINS = new Boolean(System.getenv("JENKINS")).booleanValue();
+    }
+    if (System.getenv("INGRESSPERDOMAIN") != null) {
+      INGRESSPERDOMAIN = new Boolean(System.getenv("INGRESSPERDOMAIN")).booleanValue();
+    }
+  }
+
+  /**
+   * This method gets called only once before any of the test methods are executed. It does the
+   * initialization of the integration test properties defined in OperatorIT.properties and setting
+   * the resultRoot, pvRoot and projectRoot attributes.
+   *
+   * @throws Exception
+   */
+  @BeforeClass
+  public static void staticPrepare() throws Exception {
+    // initialize test properties and create the directories
+    initialize(appPropsFile);
+  }
+
+  /**
+   * Releases k8s cluster lease, archives result, pv directories
+   *
+   * @throws Exception
+   */
+  @AfterClass
+  public static void staticUnPrepare() throws Exception {
+    logger.info("+++++++++++++++++++++++++++++++++---------------------------------+");
+    logger.info("BEGIN");
+    logger.info("Run once, release cluster lease");
+
+    StringBuffer cmd =
+        new StringBuffer("export RESULT_ROOT=$RESULT_ROOT && export PV_ROOT=$PV_ROOT && ");
+    cmd.append(BaseTest.getProjectRoot())
+        .append("/integration-tests/src/test/resources/statedump.sh");
+    logger.info("Running " + cmd);
+
+    ExecResult result = ExecCommand.exec(cmd.toString());
+    if (result.exitValue() == 0) logger.info("Executed statedump.sh " + result.stdout());
+    else
+      logger.info("Execution of statedump.sh failed, " + result.stderr() + "\n" + result.stdout());
+
+    if (JENKINS) {
+      cleanup();
+    }
+
+    if (getLeaseId() != "") {
+      logger.info("Release the k8s cluster lease");
+      TestUtils.releaseLease(getProjectRoot(), getLeaseId());
+    }
+
+    logger.info("SUCCESS");
+  }
+
+  /**
+   * Create Operator1 and domainOnPVUsingWLST with admin server and 1 managed server if they are not
+   * running. After verifying the domain is created properly change some properties on domain
+   * resources that would cause servers to be restarted and verify that servers are indeed restarted.
+   * The properties tested here are: env: "-Dweblogic.StdoutDebugEnabled=false"-->
+   * "-Dweblogic.StdoutDebugEnabled=false" logHomeEnabled: true --> logHomeEnabled: false
+   * includeServerOutInPodLog: true --> includeServerOutInPodLog: false
+   * imagePullPolicy: IfNotPresent -->  imagePullPolicy: Never
+   *
+   * @throws Exception
+   */
+  @Test
+  public void testServerRestartByDomainOnPVUsingWLST() throws Exception {
```

Comment:
change test name to testServerRestartByChangingDomainProperty or something which corresponds to what is being tested?